src/AstroImages.jl

```diff
@@ -87,8 +87,14 @@ end
 
 mutable struct Properties{P <: Union{AbstractFloat, FixedPoint}}
     rgb_image::MappedArrays.MultiMappedArray{RGB{P},2,Tuple{Array{P,2},Array{P,2},Array{P,2}},Type{RGB{P}},typeof(ImageCore.extractchannels)}
+    contrast::Float64
+    brightness::Float64
+    label::Array{Tuple{Tuple{Float64,Float64},String},1}
     function Properties{P}(;kvs...) where P
         obj = new{P}()
+        obj.contrast = 1.0
+        obj.brightness = 0.0
+        obj.label = Array{Tuple{Tuple{Float64,Float64},String}}(undef,0)
```

Comment:
I think `Tuple{Tuple{Float64,Float64},String}[]` would also work, but they should be equivalent, so there is no need to change.

Reply:
But the labels can't be changed that way, right? It would we restricting a possible functionality.